File: 1689-PartitioningIntoMinimumNumberOfDeci-BinaryNumbers/1689-PartitioningIntoMinimumNumberOfDeci-BinaryNumbers.cpp
// Last updated: 3/14/2026, 10:30:58 PM
1class Solution {
2public:
3    int minPartitions(string n) {
4        int maxDigit = 0;
5
6        for(char value : n){
7            maxDigit = max(maxDigit , value - '0');
8        }
9        return maxDigit;
10    }
11};

File: 1689-PartitioningIntoMinimumNumberOfDeci-BinaryNumbers/1689-PartitioningIntoMinimumNumberOfDeci-BinaryNumbers.js
// Last updated: 3/14/2026, 10:27:36 PM
1/**
2 * @param {string} n
3 * @return {number}
4 */
5var minPartitions = function(n) {
6    let value = n.split("").map(Number);
7    return Math.max(...value)
8};